extensions/VisualEditor/wikia/ApiAddMediaPermanent.php

```diff
@@ -28,9 +28,7 @@ private function executeImage() {
 		);
 		$duplicate = $this->getFileDuplicate( $tempFile->getLocalRefPath() );
 		if ( $duplicate ) {
```

Comment:
What do you think about converting if..else to single if, where condition will be `!$file = $duplicat`?